packages/react-interactive-tab/package.json

```diff
@@ -2,13 +2,15 @@
   "name": "@fluentui-contrib/react-interactive-tab",
   "version": "0.0.1",
   "dependencies": {
+    "@fluentui/react-jsx-runtime": ">=9.0.29 < 10.0.0",
+    "@fluentui/react-tabs": ">=9.5.0 < 10.0.0",
```

Comment:
hmm wondering if we shouldn't rather put this to peerDeps or remove completely. the peerDep requirement will force users to install tabs under the hood anyways. if we keep it as is it might introduce react tab dupes ( impacting bundle size / potential runtime errors )

also for dependencies we don't use ranges like for peerDeps, only carets and tilda `~`/`^`

Reply:
> note: for dependencies we don't use ranges like for peerDeps, only carets and tilda ~/^

As mentioned in another comment, I took it from the `react-chat` package from this repo.

Reply:
yes it does, but that's wrong ( gte lte makes sense only in peerDep specificiation )

the side effect of people copy pasting manually from generated peerDeps instead of automation generating these things. cc @ling1726